src/event-listeners/index.js

```diff
@@ -77,6 +83,56 @@ export default function setupEventListeners(store, canvas) {
           // prevent the browser from navigating back to the last page
           event.preventDefault()
         }
+      case 'ControlLeft':
```

Comment:
```suggestion
      case 'ControlLeft':
      case 'ControlRight':
```

we should handle both keyboard keys, just like with ShiftLeft and ShiftRight, just above this ^^